lib/pages/search/search_debouncer_mixin.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:debounce_throttle/debounce_throttle.dart';
+import 'package:matrix/matrix.dart';
+
+mixin SearchDebouncerMixin {
+  static const _debouncerIntervalInMilliseconds = 300;
+
+  Debouncer<String>? _debouncer;
+
+  void initializeDebouncer(void Function(String)? onDebouncerChanged) {
+    _debouncer = Debouncer(
+      const Duration(milliseconds: _debouncerIntervalInMilliseconds),
+      initialValue: '',
+    );
+
+    _debouncer?.values.listen((keyword) async {
+      Logs().d(
+        "SearchContactAndRecentChatController::_initializeDebouncer: searchKeyword: $keyword",
```

Comment:
change log to right class
or use debugLable

Reply:
done